test/ProductConstructionService.ScenarioTests/ScenarioTests/ScenarioTests_CodeFlowConflicts.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ServiceModel.Channels;
+using Microsoft.DotNet.DarcLib.VirtualMonoRepo;
+using Microsoft.DotNet.ProductConstructionService.Client.Models;
+using NUnit.Framework;
+
+namespace ProductConstructionService.ScenarioTests.ScenarioTests;
+
+[TestFixture]
+[Category("PostDeployment")]
+[Category("CodeFlow")]
+internal partial class ScenarioTests_CodeFlow : CodeFlowScenarioTestBase
+{
+    private const string CommitPlaceholder = "<commitPlaceholder>";
+    private const string ConflictMessage = $"""
+        There was a conflict in the PR branch when flowing source from https://github.com/maestro-auth-test/maestro-test1/tree/{CommitPlaceholder}
+        Conflicting files:
+         - {TestFile1Name}
+         - {TestFile2Name}
+
+        Updates from this subscription will be blocked until the conflict is resolved, or the PR is merged
+        
+        """;
```

Comment:
I don't think we should be matching the comment in its entirety. We might be changing this here and there and we won't find out about breaking this test until a main build completes. 
I'd make it more resilient and match some key parts such as "conflict" or the list of files.

Reply:
fixed, we now just check if the comment has the files and urls we expect them to be at